subprojects/android-test/test-app/gradle.properties

```diff
@@ -1,4 +1,2 @@
-android.enableR8=true
```

Comment:
Useless since AGP 3.6

```
WARNING: The option 'android.enableR8' is deprecated and should not be used anymore.
It will be removed in a future version of the Android Gradle plugin, and will no longer allow you to disable R8.
```